### DISCUSSION TITLE
edge_label_index

### DISCUSSION THREAD

**Discussion by RetrosynthesisAI** [pyg-team/pytorch_geometric#3862] [Q&A] [answered]
Hi, What is the code for creating the following: edge_label_index. I was able to use the load_edge_csv() to create the edge_index and edge_label variables, as well as use them in HeteroData().

**Comment by rusty1s** [comment, marked as answer]
The [`RandomLinkSplit`](https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/transforms/random_link_split.py) code takes care of creating `edge_label_index`. That is, it will split the original `edge_index` matrix into edges used for message passing (`edge_index`) and edges used for supervision (`edge_label_index`).
